### ISSUE TITLE
updated downloadable research plan and protocol

### ISSUE THREAD

**Issue by air2h2o** [IBM-Design/idr#100]
this content belongs on the Plan for success page: https://idr-public.netlify.com/research-in-practice/plan-for-success

[ResearchPlan_v3.docx](https://github.com/IBM-Design/idr/files/3478239/ResearchPlan_v3.docx)
[ResearchProtocol-v3.docx](https://github.com/IBM-Design/idr/files/3478240/ResearchProtocol-v3.docx)

**Comment by air2h2o** [comment]
We got an email from support that the links are coming up 404. I tested it and was getting the same 404 error. 

support email vvvv

User-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:68.0)
Gecko/20100101 Firefox/68.0
Remote-host: cpe-70-114-202-57.austin.res.rr.com 70.114.202.57
Country-code:
Referred-from: -

-- Customer information -------------------------------------

Salutation: Ms.
First name: Kara
Last name: Kotwas
E-mail address: kakotwas@us.ibm.com
Your country of region: us
Help us answer your message: Employee
Message: Both links to Design research plan and protocol templates from
this
page lead to a 404 error.

https://www.ibm.com/design/research/research-in-practice/plan-for-success#get-st


arted
comment: An error was encountered while accessing an IBM Web site (HTTP
response code 404).
Requested URL: https://www.ibm.com/design/ResearchPlan_v3.docx
Referred from: -
